### PR TITLE
Update beberlei/assert to ~3.0

### PR DESCRIPTION
**Pull request by Lutacon**
Hello,

We are using broadway but also want to have the new features from beberlei/assert. Just basically the fix `AssertionFailedException` 👀 

On 3.0 they are dropping support of PHP < 7.0

https://github.com/beberlei/assert/releases

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "keywords": ["cqrs", "event sourcing", "domain-driven design", "ddd"],
     "license": "MIT",
     "require": {
-        "beberlei/assert": "~2.0",
+        "beberlei/assert": "~3.0",
```
Comment (ruudk) [inline composer.json]: Why not allow both `~2.0 || ~3.0`. There is no real reason to only required 3.0.

**Comment by ruudk** [comment]
I added a comment to also allow 2.0. Would be nice it this can be merged as it's currently blocking us from upgrading to PHP 7.3 (since url regex is broken in Assert 2.0)

File: composer.json (hunk 1 above)
Comment (wjzijderveld) [reply composer.json]: There is not a hard reason, but also doesn't add a lot of value imo. 
We require PHP7, beberlei/assert 3.0 is released for PHP7, so I see no reason not make that the requirement.

**Comment by wjzijderveld** [comment]
Thanks @Lutacon !

**Comment by ruudk** [comment]
Great, will it be tagged as a new release?

**Pull request merged**